### PR TITLE
📖  Remove BMO vars from docs

### PR DESCRIPTION
**Pull request by kashifest**
There are still some leftover documentation which was needed when BMO was part of CAPM3. These are not needed anymore and misleading also perhaps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-[![CentOS V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/badge/icon?subject=CentOS%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_centos/)
+[![Ubuntu V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_ubuntu/badge/icon?subject=Ubuntu%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_ubuntu/)
+[![CentOS V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/badge/icon?subject=CentOS%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_centos/)
 
-This repository contains a Machine actuator implementation for the
-Kubernetes [Cluster API](https://github.com/kubernetes-sigs/cluster-api/).
+Kubernetes-native declarative infrastructure for Metal3.
```
Comment (fmuyassarov) [inline README.md:6]: This seems a bit confusing to me

File: README.md [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 
-For more information about this actuator and related repositories, see
-[metal3.io](http://metal3.io/).
+## What is the Cluster API Provider Metal3
+
+The [Cluster API](https://github.com/kubernetes-sigs/cluster-api/)brings declarative,
+Kubernetes-style APIs to cluster creation, configuration and management. The API 
+itself is shared across multiple cloud providers. Cluster API Provider Metal3 is 
+one of the providers for Cluster API and enables users to deploy a Cluster API based
+cluster on top of BareMetal infrastructure using Metal3.
```
Comment (fmuyassarov) [inline README.md:14]:
```suggestion
cluster on top of bare metal infrastructure using Metal3. 
```

File: README.md [hunk 3]
```diff
@@ -1,27 +1,28 @@
...
+one of the providers for Cluster API and enables users to deploy a Cluster API based
+cluster on top of BareMetal infrastructure using Metal3. 
 
 ## Compatibility with Cluster API
 
 | CAPM3 version | Cluster API version | CAPM3 Release |
 |---------------|---------------------|---------------|
-| ~~v1alpha3~~  | ~~v1alpha3~~        | ~~v0.3.X~~    |
 | v1alpha4      | v1alpha3            | v0.4.X        |
 | v1alpha5      | v1alpha4            | v0.5.X        |
```
Comment (fmuyassarov) [inline README.md:21]: maybe you could add another row to show that it is deprecated ?

File: README.md [hunk 4]
```diff
@@ -1,27 +1,28 @@
 # Cluster API Provider for Managed Bare Metal Hardware
 
-[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_ubuntu/badge/icon?subject=Ubuntu%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_ubuntu/)
-[![CentOS V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/badge/icon?subject=CentOS%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_centos/)
+[![Ubuntu V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_ubuntu/badge/icon?subject=Ubuntu%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_ubuntu/)
```
Comment (furkatgofurov7) [inline README.md:3]:
```suggestion
[![Ubuntu V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_ubuntu/badge/icon?subject=Ubuntu%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_ubuntu/)
```

File: README.md [hunk 5]
```diff
@@ -1,27 +1,28 @@
...
+[![CentOS V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/badge/icon?subject=CentOS%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a4_integration_test_centos/)
```
Comment (furkatgofurov7) [inline README.md:4]:
```suggestion
[![CentOS V1alpha5 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/badge/icon?subject=CentOS%20E2E%20V1alpha5)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a5_integration_test_centos/)
```

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Comment by furkatgofurov7** [comment]
> But I'm afraid tide is not going to merge it. Let's see :)

Just a wild guess, maybe tide pod has to be re-deployed in prow cluster?

**Pull request closed without merging**